reactnative/RNPlayground/app/screens/PlaygroundScreen.tsx

```diff
@@ -1,76 +1,108 @@
-import React, { FC } from "react"
+import * as React from "react"
 import { observer } from "mobx-react-lite"
-import { StackScreenProps } from "@react-navigation/stack"
-import { AppStackScreenProps } from "../navigators"
-import { Image, ImageStyle, TextStyle, View, ViewStyle } from "react-native"
-import {
-  Text, Screen
-} from "../components"
-import { isRTL } from "../i18n"
```

Comment:
This is fine to go as its not really used, but incase it comes up again - this returns if the current device language direction is `Right-To-Left`. This is part of the `I18nManager` localization lib, which supports multiple languages including Arabic.

Reply:
it's weird that this would need to be in the feature code though